### ISSUE TITLE
DOC - Improved headings formatting

### ISSUE THREAD

**Issue by hamishwillee** [DroidPlanner/DroneKit-Android#106]
Sphinx doesn't care what underline characters you use to mark up headings, as long as they are longer than the preceding text. 

However it is usually easier to read and edit the markup if you use the following underline markup pattern for the different levels and have a space before the heading (except for title)

```
================
Document Title 
================

A first level heading
=====================

A second level heading
----------------------------------------

 etc
```

This is because = is the easiest character for your eye to scan as a heading line.

**Comment by hamishwillee** [comment]
I've updated the docs in line with this in #105

**Comment by ne0fhyk** [comment]
Fixed by #105
